Makefile

```diff
@@ -8,6 +8,11 @@ SNAPSHOT?=yes
 PYTHON_ENV?=${BUILD_DIR}/python-env
 VIRTUALENV_PARAMS?=
 FIND=find . -type f -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*"
+GOLINT=golint
```

Comment:
Ok for me. My main worry is that we have quite a few duplicated lines and will change them in the future in one place but not the other.